docs/tutorial/getting-started.rst

```diff
@@ -112,6 +112,22 @@ iOS
   install the Xcode Command Line Tools. This can be done from the Preference
   panel within XCode itself.
 
+* For first time XCode users, make sure you've installed or updated to the latest XCode version.
+
+* You'll need to sign in to XCode with your Apple ID account - the same one you'd use for iTunes.
+
+* On Mohave, the easiest way to ensure that the Xcode Command Line Tools are
+  installed is to open a default project template in XCode and run for the newest version of iPhone.
```

Comment:
Is this method preferable to `xcode-select --install` from the terminal? I think going through the Xcode UI we might have to explain more steps (what is a project? How does one run it?)